### PR TITLE
fix(README): fix command for installing

### PR DESCRIPTION
**Pull request by adelkahomolova**
fix: name of contributor
feat: add aliases to readme,

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,16 @@ dxscanner ~/my-project
 ```
 dxscanner [path]
```
Comment (prokopsimek) [inline README.md]: Set `dx-scanner` as default command to be consistent

File: README.md [hunk 2]
```diff
@@ -70,7 +80,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
 <table>
   <tr>
     <td align="center"><a href="https://github.com/prokopsimek"><img src="https://avatars2.githubusercontent.com/u/5487217?v=4" width="100px;" alt="Prokop Simek"/><br /><sub><b>Prokop Simek</b></sub></a><br /><a href="https://github.com/DXHeroes/dx-scanner/commits?author=prokopsimek" title="Code">💻</a> <a href="#maintenance-prokopsimek" title="Maintenance">🚧</a></td>
-    <td align="center"><a href="https://github.com/adelkahomolova"><img src="https://avatars2.githubusercontent.com/u/53510747?v=4" width="100px;" alt="adelkah"/><br /><sub><b>adelkah</b></sub></a><br /><a href="https://github.com/DXHeroes/dx-scanner/commits?author=adelkahomolova" title="Code">💻</a> <a href="#maintenance-adelkahomolova" title="Maintenance">🚧</a></td>
+    <td align="center"><a href="https://github.com/adelkahomolova"><img src="https://avatars2.githubusercontent.com/u/53510747?v=4" width="100px;" alt="adelkah"/><br /><sub><b>Adela Homolova</b></sub></a><br /><a href="https://github.com/DXHeroes/dx-scanner/commits?author=adelkahomolova" title="Code">💻</a> <a href="#maintenance-adelkahomolova" title="Maintenance">🚧</a></td>
```
Comment (prokopsimek) [inline README.md]: This is autogenerated from `.all-contributorsrc`, you have to change it there or event better on Github.

File: README.md (hunk 1 above)
Comment (adelkahomolova) [reply README.md]: I was thinking about this actually :D Will do it :)

File: README.md (hunk 2 above)
Comment (adelkahomolova) [reply README.md]: Ok, there are more important tasks now. I will discard this change fo now and will change it when I'll have a time.

**Pull request merged**